### DISCUSSION TITLE
A problem with enhance sampling

### DISCUSSION THREAD

**Discussion by Tgernew** [deepmodeling/dpgen#1135] [Q&A]
DPGEN version 0.11.0
Deepmd-kit version v2.1.1
I use the similar input with the examples and here is my model_devi input,

  
        
 
        {   "sys_idx": [0],"trj_freq": 10,"_idx": "00",

            "template":{ "lmp": "lmp/input.lammps", "plm": "lmp/plumed.dat" },

            "rev_mat":{ "lmp": {"V_NSTEPS": [20000], "V_TEMP": [300], "V_PRES": [1]},

                        "plm": {"V_TEMP":  [300], "V_STRIDE": [10]}

                      }
        },

but something went wrong during the model_devi process, the model_devi.log gives the error:

  >>> Info of lammps module:
  use deepmd-kit at:  /public/home/elecorrosion/.conda/envs/deepmd2023-02-07 20:30:10.471206: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
  >>> Info of model(s):
  using   4 model(s): ../graph.000.pb ../graph.001.pb ../graph.002.pb ../graph.003.pb
  rcut in model:      6
  ntypes in model:    3
+++ Loading the PLUMED kernel runtime +++
+++ PLUMED_KERNEL="/public/home/elecorrosion/.conda/envs/deepmd/lib/libplumedKernel.so" +++
ERROR: Incompatible API version for PLUMED in fix plumed. Only Plumed 2.4.x, 2.5.x, 2.6.x, 2.7.x are tested and supported. (src/PLUMED/fix_plumed.cpp:81)
Last command: fix            dpgen_plm all plumed plumedfile input.plumed outfile output.plumed
PLUMED instances was not properly deallocated in your code: 1

I'm sure both lammps and plumed were installed along with DPGEN, I'm wonder what can I do to fix this error? should I reinstalled DPGEN?

**Comment by AnguseZhang** [comment]
I think this problem is related to DeePMD-kit when loading Plumed plugins. Maybe you can find what's wrong by going into the folders where it runs.

**Reply by Tgernew** [reply]
Thanks alot, the problem has been solved. a previously installed plumed caused an error while it's running.
